tests/live/README.md

```diff
@@ -0,0 +1,8 @@
+# Live
+
+This is the super integration test, because we spend real testnet ETH to do this. The reason, is that we need to test verification. 
```

Comment:
Sepolia and testnet are both used, which can be a bit confusing as people might think they are different things

```suggestion
This is an integration test, because we spend Sepolia ETH to do this.
The reason is for testing contract verification. 
```